src/pages/Group/Group.js

```diff
@@ -1,5 +1,392 @@
+                                        isFloating={true}
+                                        />
+                                    </div>
+                                    <div className='cuisine'>
+                                        <FilterButton content={
+                                            <p>Cuisine</p>
+                                        } 
+                                        onClickHandler={() => handleSingleBtnColor(filterBtnStyles, setFilterBtnStyles, 0)}
+                                        onFilter={filterBtnStyles[0]}
+                                        />
+                                    </div>
+                                    <div className='priceRange'>
+                                        <FilterButton content={
+                                            <p>Price Range</p>
+                                        } 
+                                        onClickHandler={() => handleSingleBtnColor(filterBtnStyles, setFilterBtnStyles, 1)}
+                                        onFilter={filterBtnStyles[1]}
+                                        />
+                                    </div>
+                                    <div className='allergens'>
+                                        <FilterButton content={
+                                            <p>Diet and Allergens</p>
+                                        } 
+                                        onClickHandler={() => handleSingleBtnColor(filterBtnStyles, setFilterBtnStyles, 2)}
+                                        onFilter={filterBtnStyles[2]}
+                                        />
+                                    </div>
+                                    <div className='mealType'>
+                                        <FilterButton content={
+                                            <p>Meal Type</p>
+                                        } 
+                                        onClickHandler={() => handleSingleBtnColor(filterBtnStyles, setFilterBtnStyles, 3)}
+                                        onFilter={filterBtnStyles[3]}
+                                        />
+                                    </div>
+                                </div>
+                                {filterBtnStyles[0] ? cuisineCards() : null}
+                                {filterBtnStyles[1] ? priceRangeCards() : null}
+                                {filterBtnStyles[2] ? allergensCards() : null}
+                                {filterBtnStyles[3] ? mealCards() : null}
+                            </div>
+                        }/>
+                    </div>
+                    <div className='rightCardContainer'>
+                        <Card content={
+                            <div className='rightCardContent'>
+                                <h2 className='groupMemberTitle'>Group members: </h2>
+                                <div className='memberProfile'>
```

Comment:
these you can probably abstract into their own component as well or convert them into a map (that iterates through like a list of names) so you don't have to repeat them